clock/app/views/employees/_form.html.erb

```diff
@@ -29,67 +30,77 @@
+          <%= form.label :position, class: "col-sm-2 col-form-label" %>
           <div class="col-sm-10">
-          <%= form.text_field :position, placeholder: "CEO" %>
+          <%= form.text_field :position, placeholder: "CEO", class: "form-control" %>
           </div>
         </div>
 
         <div class="form-group row">
-          <%= form.label :phone, :class => "col-sm-2 col-form-label" %>
+          <%= form.label :phone, class: "col-sm-2 col-form-label" %>
           <div class="col-sm-10">
-          <%= form.text_field :phone, placeholder: "+52 (312) 000 0000" %>
+          <%= form.text_field :phone, placeholder: "+52 (312) 000 0000", class: "form-control" %>
           </div>
         </div>
 
         <div class="form-group row">
-          <%= form.label :address, :class => "col-sm-2 col-form-label" %>
+          <%= form.label :address, class: "col-sm-2 col-form-label" %>
           <div class="col-sm-10">
-          <%= form.text_field :address, placeholder: "Address Name # 101, Colima" %>
+          <%= form.text_field :address, placeholder: "Address Name # 101, Colima", class: "form-control" %>
           </div>
         </div>
 
         <div class="form-group row">
-          <%= form.label :nip, :class => "col-sm-2 col-form-label" %>
+          <%= form.label :nip, class: "col-sm-2 col-form-label" %>
           <div class="col-sm-10">
-          <%= form.text_field :nip, placeholder: "1234" %>
+          <%= form.text_field :nip, placeholder: "1234", class: "form-control" %>
           </div>
         </div>
 
         <div class="form-group row">
-          <%= form.label :work_days, :class => "col-sm-2 col-form-label" %>
+          <%= form.label :work_days, class: "col-sm-2 col-form-label" %>
           <div class="col-sm-10 row">
           <% ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'].each do |work_day| -%>
-            <%= check_box_tag "employee[work_days][#{work_day}]", work_day, @employee.work_days&.any?(work_day), :class => "col-sm-2 col-form-label", :checked => true  %>
+            <%= check_box_tag "employee[work_days][#{work_day}]", work_day, @employee.work_days&.any?(work_day), class: "col-sm-2 col-form-label", checked: true  %>
             <%= label_tag "employee[work_days][#{work_day}]", work_day, :class => "col-sm-2 col-form-label"  %>
           <% end -%>    
           </div>
         </div>
 
         <div class="form-group row">
-          <%= form.label "Work  hours", :class => "col-sm-2 col-form-label" %>
+          <%= form.label "Work  hours", class: "col-sm-2 col-form-label" %>
           <div class="col-sm-10">
           <%= form.time_select :enter_time %> &nbsp; to &nbsp; <%= form.time_select :departure_time %>
           </div>
         </div>
 
         <div class="form-group row">
           <div class="col-sm-10 offset-sm-2">
-            <%= form.submit class: 'btn btn-primary float-right' %>
+          <% if @show_btn_employee %>
```

Comment:
I see what you are doing here, and it works, but something seems wrong.... I mean, it looks hacky

Partial should not be handling this kind of logic, it's intended to only contain shared elements between your `new` and `edit` views

I think is kind of difficult changing the structure here....
And also, I just noticed that you already have the `destroy` action in the index, so, perhaps this is not needed at all....


I'm not requesting any change, just sharing my thougths, sorry for not providing any valuable feedback, hahah 😅

Reply:
Any comment is valuable. We implement it like this, because the delete button will only appear in the view to update the information of each employee.

Reply:
Hi @YilKanda, you can pass one argument to the `render partial: 'form', locals: { show_delete_btn: true }`, this way you can avoid using the instance variable.

The downside of this approach is that you need to pass the `show_delete_btn` in every view you render the partial, but I'll say is still worth it.